backend/app/controllers/api/v1/sessions_controller.rb

```diff
@@ -0,0 +1,25 @@
+module API
+  module V1
+    class SessionsController < BaseController
+      before_action :authenticate_user!, only: [:destroy]
+
+      def create
+        user = User.find_by_email(params[:email])
+
+        if user&.valid_password?(params[:password])
+          # warden checks devise options like confirmable
+          # if not confirmed then throws an error that is handled by failure_app
+          sign_in user
+          render json: UserSerializer.new(user).serializable_hash
+        else
+          raise API::UnprocessableEntityError, "Invalid email or password"
+        end
+      end
+
+      def destroy
+        sign_out current_user
+        render json: {data: nil}
```

Comment:
maybe return of `head :ok` could be enough

Reply:
Hmm, yeah, I will be simpler. We can do it for all similar endpoints that perform some action but no response is needed.